readme.md

```diff
@@ -5,22 +5,41 @@
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 [![Join the chat at https://gitter.im/LearningLocker/learninglocker](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/LearningLocker/learninglocker?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-*Learning Locker is a trademark of [HT2 Inc.](http://ht2labs.com)*
+*Learning Locker is a trademark of [Learning Pool](http://learningpool.com)*
 
-### Installation
+### Development: Installation
+
+1. [Install the latest version of Git for version control.](https://git-scm.com/downloads)
+1. [Install the latest version of N for managing Node and NPM versions.](https://github.com/tj/n#third-party-installers)
+1. [Install the latest version of Docker.](https://docs.docker.com/get-docker/)
+1. Install the latest version of yarn for dependency management with `npm i -g yarn`.
+1. Clone the repository with `git clone git@github.com:LearningLocker/xapi-service.git`.
+1. Switch to the repository directory with `cd xapi-service`.
+1. Install dependencies with `yarn`.
+1. Copy the ".env.example" file in the repository into a new ".env" file.
+1. Build the code `yarn build` or `yarn build --watch` if you want to make changes.
+1. Start Mongo and Redis with `docker-compose up -d`. If you've followed the [Enterprise setup instructions](https://github.com/LearningLocker/enterprise/blob/master/README.md) already you won't need to do this.
+1. Run the server with `yarn start`.
+
+### Development: Testing
+
+Before you follow these instructions you may want to exit your `yarn start` command above with Ctrl + C. This ensures that the running app doesn't interfere with your testing by using Mongo and Redis.
+
+1. Install dependencies with `yarn`.
+1. Start Mongo and Redis with `docker-compose up -d`. If you've followed the [Enterprise setup instructions](https://github.com/LearningLocker/enterprise/blob/master/README.md) already you won't need to do this.
+1. Lint the code with `yarn lint`.
+1. Build the code with `yarn build`.
+1. Test the code with `yarn cover-ci`.
+1. Stop the Mongo and Redis with `docker-compose down`. Use `-v` at the end to delete data.
+
+### Production: Installation
 To install all of Learning Locker, see the [installation documentation](http://docs.learninglocker.net/guides-installing/). To install just the xAPI service, you can follow the instructions below.
 
-1. Clone the repository `git clone git@github.com:LearningLocker/xapi-service.git`.
-1. Install dependencies `npm install`.
-1. Build the code `npm run build`.
-1. Start the server `npm start`.
-
-### Development
```

Comment:
Did you intend to remove the Dev section @ryansmith94 ?

Also, is it worth noting the yarn start:dev option (I guess it's easily found using the package.json to locate the repo command list).

Reply:
Yeah I split into two Installation and Testing.

Yes good shout, I've added that option 👍